config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -227,6 +253,49 @@ periodics:
     testgrid-tab-name: gce-windows-containerd-master
     description: Runs tests on a Kubernetes cluster with Windows containerd nodes on GCE
 
+- name: ci-kubernetes-e2e-windows-node-throughput
+  interval: 2h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-common-gce-windows: "true"
+    preset-load-gce-windows: "true"
+  annotations:
+    testgrid-dashboards: google-windows
+    testgrid-tab-name: windows-gce-node-throughput
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: windows-testing
```

Comment:
Thanks for updating this. The windows-testing repo has a slightly different layout than gce-k8s-windows-testing: the relevant code is in the gce/ subdir of the new repo. Did you make that change + validate it?

Reply:
checkout the repo and validate locally, after adding executable permission to load-test.sh, it's working fine.